src/Provider/Provider.Web/Orchestrators/DeleteVacancyOrchestrator.cs

```diff
@@ -48,7 +48,14 @@ public async Task DeleteVacancyAsync(DeleteEditModel m, VacancyUser user)
             if (!vacancy.CanDelete)
                 throw new InvalidStateException(string.Format(ErrorMessages.VacancyNotAvailableForEditing, vacancy.Title));
 
+            var vm = new DeleteViewModel
+            {
+                Title = vacancy.Title,
+                Status = vacancy.Status,
+                VacancyReference = vacancy.VacancyReference
+            };
             await _client.DeleteVacancyAsync(vacancy.Id, user);
+            return vm;
```

Comment:
unit test please to confirm that this view model is returned and contains the correct values